mimic/model/rackspace_image_store.py

```diff
@@ -0,0 +1,66 @@
+"""
+An image store representing Rackspace specific images
+"""
+from __future__ import absolute_import, division, unicode_literals
+from characteristic import attributes, Attribute
+from six import iteritems
+from mimic.model.rackspace_images import (RackspaceWindowsImage, RackspaceArchImage,
+                                          RackspaceCentOSPVImage, RackspaceCentOSPVHMImage,
+                                          RackspaceCoreOSImage, RackspaceDebianImage,
+                                          RackspaceFedoraImage, RackspaceFreeBSDImage,
+                                          RackspaceGentooImage, RackspaceOpenSUSEImage,
+                                          RackspaceRedHatPVImage, RackspaceRedHatPVHMImage,
+                                          RackspaceUbuntuPVImage, RackspaceUbuntuPVHMImage,
+                                          RackspaceVyattaImage, RackspaceScientificImage,
+                                          RackspaceOnMetalCentOSImage, RackspaceOnMetalCoreOSImage,
+                                          RackspaceOnMetalDebianImage, RackspaceOnMetalFedoraImage,
+                                          RackspaceOnMetalUbuntuImage)
+
+
+@attributes([Attribute("image_list", default_factory=list)])
+class RackspaceImageStore(object):
+    """
+    A store for images to share between nova_api and glance_api
+    :var image_list: list of Rackspace images
+    """
+    def create_image_store(self, tenant_id):
+        """
+        Generates the data for each image in each image class
```

Comment:
This docstring also neglects to mention that the method has a return value, and what that return value is.